public_html/assets/js/custom.js

```diff
@@ -175,3 +175,25 @@ $(window).resize(function () {
     console.log('sidebar = ' + $('#SS_sidebar').width());
     console.log('chart = ' + $('#SS_piechart').width());
 });
+
+
+//navigation menu for small screens
+$(function() {
+    var pull        = $('#nav-pull');
+        menu        = $('#navigation ul');
+        menuHeight  = menu.height();
```

Comment:
Evo, bom tokrat jz špilu unga ta tečnega modela ki opozarja na male napakice :)

Tole bi verjetno moralo biti:

    var pull        = $('#nav-pull'),
        menu        = $('#navigation ul'),
        menuHeight  = menu.height();

Če imamo `;` in ne `,` mimogrede na tiho naredimo [implicit global spremenljivki](http://blog.niftysnippets.org/2008/03/horror-of-implicit-globals.html) `menu` in `menuHeight`, kar ni najbolj lepo. 

Če želimo uporabljati globalne spremenljivke, jih je zelo 'vljudno' uporabljati eksplicitno, torej `window.menu` ali `window.menuHeight`.